common/org.testeditor.dsl.common.testing/build.properties

```diff
@@ -2,4 +2,5 @@ source.. = src/,\
            xtend-gen/
 output.. = bin/
 bin.includes = META-INF/,\
-               .
+               .,\
+               lib/mockito-all-1.10.19.jar
```

Comment:
I would prefer it, to use maven to download and copy the dependency to the lib. This is an example from:
`		<plugin>
				<artifactId>maven-dependency-plugin</artifactId>
				<executions>
					<execution>
						<phase>process-sources</phase>

						<goals>
							<goal>copy-dependencies</goal>
						</goals>

						<configuration>
							<outputDirectory>lib</outputDirectory>
							<includeArtifactIds>core-fixture,log4j-core,log4j-api</includeArtifactIds>
						</configuration>
					</execution>
				</executions>
			</plugin>`
from: https://github.com/test-editor/test-editor-xtext/blob/develop/rcp/org.testeditor.rcp4.uatests/pom.xml